backend/src/main/java/com/tourneynizer/tourneynizer/model/UserInfo.java

```diff
@@ -4,10 +4,27 @@
 
     public int wins, losses, tournaments, matches;
 
-    public UserInfo(int wins, int losses, int tournaments, int matches) {
+    public UserInfo(int wins, int losses, int tournaments) {
         this.wins = wins;
         this.losses = losses;
         this.tournaments = tournaments;
-        this.matches = matches;
+        this.matches = wins + losses;
+    }
+
+    public UserInfo() {
+        this(0, 0, 0);
+    }
+
```

Comment:
How will spring convert this to JSON? Do you need getters or since the members are public you don't need to?

Reply:
It works because they're public. Look at `UserInfoTest#json` for more exact info.

Reply:
`UserTest#json`, sorry